index.html

```diff
@@ -58,6 +58,10 @@ <h2>Jason</h2>
                     <h2>Jacob</h2>
                     <a href="http://www.x.com/arcadejacob"><img class="profile" src="images/jacob.jpg" alt="Profile photo of Jacob Ashley"></a>
                 </section>
+                <section>
+                    <h2>Ellie</h2>
+                    <a href="http://www.x.com/elliezub"><img class="profile" src="images/ellie.jpg" alt="Profile photo of Ellie"></a>
+                </section>
```

Comment:
Jason Torres & Jacob Ashley - should be changed to Ellie as well

Reply:
Do you mean there is only 1 host ? Hope I am understanding it correctly ?? @shivaylamba

Reply:
Done @shivaylamba